netbox_data_import/templates/netbox_data_import/importexecution_list.html

```
@@ -1,5 +1,5 @@
 {# SPDX-License-Identifier: Apache-2.0 #}
-{# Copyright (C) 2025 Marcin Zieba <marcinpsk@gmail.com> #}
+{# SPDX-FileCopyrightText: 2026 Marcin Zieba <marcinpsk@gmail.com> #}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)